cmd/cstor-pool-mgmt/volumereplica/volumereplica.go

```diff
@@ -214,6 +214,28 @@ func parseCapacityUnit(capacity string) string {
 	return capacity
 }
 
+// Capacity finds the capacity of the volume.
+// The ouptut of command executed is as follows:
+/*
+root@cstor-sparse-pool-6dft-5b5c78ccc7-dls8s:/# zfs get used,logicalused cstor-d82bd105-f3a8-11e8-87fd-42010a800087/pvc-1b2a7d4b-f3a9-11e8-87fd-42010a800087
+NAME                                                                                 PROPERTY     VALUE  SOURCE
+cstor-d82bd105-f3a8-11e8-87fd-42010a800087/pvc-1b2a7d4b-f3a9-11e8-87fd-42010a800087  used         6K     -
+cstor-d82bd105-f3a8-11e8-87fd-42010a800087/pvc-1b2a7d4b-f3a9-11e8-87fd-42010a800087  logicalused  6K     -
+*/
+func Capacity(volName string) (*apis.CStorVolumeCapacityAttr, error) {
+	capacityVolStr := []string{"get", "used,logicalused", volName}
+	stdoutStderr, err := RunnerVar.RunCombinedOutput(VolumeReplicaOperator, capacityVolStr...)
+	if err != nil {
+		glog.Errorf("Unable to get volume capacity: %v", string(stdoutStderr))
+		return nil, err
+	}
+	poolCapacity := capacityOutputParser(string(stdoutStderr))
+	if strings.TrimSpace(poolCapacity.TotalAllocated) == "" || strings.TrimSpace(poolCapacity.Used) == "" {
+		return nil, fmt.Errorf("Unable to get volume capacity from capacity parser")
```

Comment:
nit_: ("failed to parse capacity from the response : %v ", poolCapacity) ?

Reply:
Message is almost same. Have used this style in previously. So I would prefer to keep the message unless the message provide wrong/confusing semantics.
But yes one nit pick is -- we are returning the error an hence first letter of log message should be small case and I will do this.